scripts/compare_benchmarks.py

```diff
@@ -66,26 +73,37 @@ def calculate_and_format_p_value(old, new):
     new_data = json.load(new_file)
 
 table_data = []
-table_data.append(["Benchmark", "prev. iter/s", "runs", "new iter/s", "runs", "change", "p-value (significant if <" + str(p_value_significance_threshold) + ")"])
+table_data.append(["Benchmark", "prev. iter/s", "runs", "new iter/s", "runs", "change [%]", "change factor", "p-value (significant if <" + str(p_value_significance_threshold) + ")"])
 
 average_diff_sum = 0.0
+changes = []
 
 for old, new in zip(old_data['benchmarks'], new_data['benchmarks']):
```

Comment:
(random line)

I like the general idea of having a "symmetric" metric - and I was originally in favour of the "change factors" proposed here. Now I am wondering: Since the days of improving TPC-H by orders of magnitude are likely over, I wonder if we really want a "main" metric which is likely to always starts with "1.0*" or in good cases with "1.1*" - this would make solid improvements look negligible... But probably this is okay, and we'll just need to get used to/learn to read these numbers.

Unless we come with a better idea now-ish I'd say we merge this PR as a incremental improvement and maybe further discuss this in person and on a whiteboard?